### PR TITLE
refact: private key configs

### PR DESCRIPTION
**Pull request by huwenchao**

### CODE REVIEW

File: offchain-modules/packages/app-multisign-server/src/sigServer.ts [hunk 1]
```diff
@@ -68,18 +68,19 @@ export class SigServer {
...
       const ethKeys = new Map<string, string>();
-      ForceBridgeCore.config.eth.multiSignKeys.forEach((key) => {
-        ethKeys[key.address] = key.privKey;
-      });
+      const ethPrivateKey = ForceBridgeCore.config.eth.privateKey;
+      const ethAddress = privateKeyToEthAddress(ethPrivateKey);
+      ethKeys[ethAddress] = ethPrivateKey;
       SigServer.keys['eth'] = ethKeys;
     }
+    logger.info(`sigKeys: ${JSON.stringify(SigServer.keys, null, 2)}`);
```
Comment (homura) [inline offchain-modules/packages/app-multisign-server/src/sigServer.ts:83]: we should **NOT** print private key info in logger

Comment (homura) [inline offchain-modules/packages/app-multisign-server/src/sigServer.ts:83]: we should NOT print private key in logger

Comment (huwenchao) [reply offchain-modules/packages/app-multisign-server/src/sigServer.ts:83]: it is used to debug, forget to delete, fixed.

**Comment by homura** [comment]
LGMT

**Pull request merged**